pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -29,6 +29,7 @@ import (
 	"k8s.io/component-helpers/scheduling/corev1/nodeaffinity"
 	"k8s.io/kubernetes/pkg/features"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
+	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/feature"
```

Comment:
This is a big no.

A better solution is to have this file define `VolumeBindingArgsValidationOptions`. Somewhat similar to this https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#new-enum-value-in-existing-field

Reply:
updated